projects/laji/src/app/shared-modules/own-submissions/own-datatable/own-datatable.component.ts

```diff
@@ -132,31 +137,31 @@ export class OwnDatatableComponent implements OnInit, AfterViewChecked, OnDestro
     {prop: 'publicityRestrictions', mode: 'large'}
   ];
   allRows: RowDocument[] = [];
-  visibleRows: RowDocument[];
-  filterBy: string;
+  visibleRows: RowDocument[] = [];
+  filterBy = '';
   selectionType: SelectionType;
-  selectedLabel: string;
+  selectedLabel = '';
   labelLoading = false;
 
-  displayMode: string;
+  displayMode = 'medium';
   defaultSort: any;
 
-  usersId: string;
-  usersIdSub: Subscription;
+  usersId = '';
+  usersIdSub!: Subscription;
 
-  subscriptionDeleteOwnDocument: Subscription;
+  subscriptionDeleteOwnDocument!: Subscription;
 
-  downloadedDocumentId: string;
+  downloadedDocumentId = '';
```

Comment:
Instead of setting a default value, you could make the variable optional `downloadedDocumentId?: string` or set the type as `downloadedDocumentId: string | undefined`